app/scripts/main.js

```diff
@@ -28,53 +30,51 @@
 
     cameraManager.onframe = function() {
       // There is a frame in the camera, what should we do with it?
- 
-      var imageData = cameraManager.getImageData();
-      var detectedQRCode = qrCodeManager.detectQRCode(imageData, function(url) {
-        if(url !== undefined) {
-          qrCodeManager.showDialog(url);
-        }
-      });
+      if (qrCodeManager.numActiveThreads < qrCodeManager.maxThreads) {
```

Comment:
Just a correction, I'd be 2.5ms on the best case if it was 4 threads - 
correct calc is 10ms / numOfThreads